src/components/structures/RightPanel.js

```diff
@@ -31,7 +31,6 @@ module.exports = React.createClass({
     displayName: 'RightPanel',
 
     propTypes: {
-        userId: React.PropTypes.string, // if showing an orphaned MemberInfo page, this is set
```

Comment:
oh I mean it should really listen to the RVS like everything else. I shall comment